example/client/include/IMessageConsumer.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
```

Comment:
please add the copyright / license paragraph.

Reply:
fixed